src/library/autodj/dlgautodj.cpp

```diff
@@ -283,6 +291,16 @@ void DlgAutoDJ::toggleAutoDJButton(bool enable) {
         default:
             break;
     }
+
+    // Update the label to display the configured transition unit
+    // just in case it was changed in the preferences dialog
+    QString transitionUnit;
```

Comment:
Please, do not duplicate this code, make a method with its content.
This is specially important since it includes a translation, and you would be forcing to translate it twice.

Reply:
[Done](https://github.com/mixxxdj/mixxx/pull/2746/commits/c2f9ad507c0407d0d8ab45b0403873cafd6fba8a). Moved duplicate code to a function.